uint64 timestamp
uint64 timestamp_sample
uint32 accel_device_id
uint32 gyro_device_id
float32[3] delta_angle
float32[3] delta_velocity
float32 delta_angle_dt
float32 delta_velocity_dt
uint8 delta_angle_clipping
uint8 delta_velocity_clipping
uint8 accel_calibration_count
uint8 gyro_calibration_count
